proposals/0005-webauthn.md

```
@@ -0,0 +1,292 @@
+5. [Design Details](#design-details)
+6. [Technical Details](#technical-details)
+7. [Drawbacks](#drawbacks)
+8. [Advantages](#advantages)
+9. [Open Questions](#open-questions)
+10. [Future Directions](#future-directions)
+
+<hr />
+
+# Summary
+
+[summary]: #summary
```

markdownlint-cli2 (0.21.0)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "summary"

(MD053, link-image-reference-definitions)